### ISSUE TITLE
Gesture Issue & No update on cart 

### ISSUE THREAD

**Issue by Ankit180898** [ibsatfatima/flutter-localization#1]
When clicked on grid item it doesn't show any feel of being tapped since it uses gesture detector we can use Inkwell instead. Also there no update on the number of items on the cart item, if the main focus is only on localization then its great, its works fine.

**Comment by ibsatfatima** [comment]
Yes main focus is localization, and thanks for raising the issue I will fix it and update the repo.
